.github/workflows/generate-models.yml

```diff
@@ -0,0 +1,67 @@
+name: Generate API models
+
+on:
+  schedule:
+    - cron:  '0 1 * * *'
+
+jobs:
+  generate:
+    name: Generate
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Install
+        uses: bahmutov/npm-install@v1
+
+      - name: Generate API models
+        run: npm run generate-models
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # The schedule event can be delayed during periods of high loads. 
+          # Therefore, we need to buffer around 2 hours.
+          # Docs: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch
+          LOOKBACK_HOURS: 26 
+
+      - name: Lint
+        run: npm run lint:fix
+
+      - name: Test
+        run: npm t
+
+      - name: Test integration
+        run: npm run test:integration
+        env:
+          API_URL: https://sellingpartnerapi-eu.amazon.com
+          API_REGION: eu-west-1
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          CLIENT_ID: ${{ secrets.CLIENT_ID }}
+          CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+          REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
+          ROLE_ARN: ${{ secrets.ROLE_ARN }}
+
+      - name: Build
+        run: npm run build
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: 'feat: update api models'
```

Comment:
I think we should use `chore: ` prefix, because otherwise every little change will trigger a version upgrade.

Reply:
@moltar : It sometimes occurs `BREAKING CHANGE` as well. Such as: there is a deprecated model.
Should we use `BREAKING CHANGE` prefix?

Reply:
I think the original repo does not have any indicators anywhere about this. So the bot can never know. I think it will up to us, humans, to decide.

I propose use `chore` prefix for all commit messages. But then squash the PRs, with more descriptive prefixes, like `feat:` or `fix:` or `docs:`. And the body of the PR will have links to original commits.